controllers_v2/keysend.ctrl.go

```diff
@@ -125,8 +126,13 @@ func (controller *KeySendController) MultiKeySend(c echo.Context) error {
 		result.Keysends = append(result.Keysends, KeySendResult{
 			Keysend: res,
 		})
+		singleSuccesfulPayment = true
 	}
-	return c.JSON(http.StatusOK, result)
+	status := http.StatusOK
+	if !singleSuccesfulPayment {
+		status = http.StatusInternalServerError
```

Comment:
is this a correct HTTP status code?
internal server error indicates it's an exception an unexpected bug in our system

payments might fail because the recipient can not receive anything. 

maybe a 422 - Unprocessable Entity?

Reply:
That would deserve it's own PR as we would need to change it everywhere we're making a payment. I'd rather have it consistent for now.

Reply:
we return a 500 if a payment fails? damn, also not a good http cititzen.